specs/_features/eip7805/fork-choice.md

```diff
@@ -324,7 +324,8 @@ def on_block(store: Store, signed_block: SignedBeaconBlock) -> None:
     # Add block timeliness to the store
     seconds_since_genesis = store.time - store.genesis_time
     time_into_slot_ms = seconds_to_milliseconds(seconds_since_genesis) % SLOT_DURATION_MS
-    attestation_threshold_ms = get_slot_component_duration_ms(ATTESTATION_DUE_BPS)
+    epoch = get_current_store_epoch(store)
```

Comment:
is this the correct way to get the epoch?

Reply:
Ah yes, I believe this is correct. And outside of fork-choice code we'd use [`compute_epoch_at_slot`](https://github.com/ethereum/consensus-specs/blob/master/specs/phase0/beacon-chain.md#compute_epoch_at_slot).